task_01.py

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""Getting four pieces of information about a single customer"""
+
+
+import data
+
+
+def sum_orders(customers, orders):
+    """This function will calculate customer orders.
+
+    Args:
+        It takes two arguments; customers and orders.
+
+    Returns:
+        A dictionary
+
+    Examples:
+        >>>
+        {'email': 'evorsoisson@komarr.net', 'total': 1287, 'orders': 3,...)
+    """
+    new_dict = {}
+    for outer_key, outer_value in customers.iteritems():
+        new_key = outer_key
+        new_value = outer_value
+        temp_dict = {}
+        real_orders = 0
+        total_sum = 0
+        for inner_value in orders.itervalues():
```

Comment:
A double loop doesn't help you here -- what you want to do is access `customers[]` using the values found in orders.